shared/engine/errors.js

```diff
@@ -1,41 +0,0 @@
-// @flow
```

Comment:
not wrapping this stuff anymore. instead actions/reducers should deal with error code transformations